cmd/mist-api-connector/mist-api-connector.go

```diff
@@ -52,6 +50,9 @@ func main() {
 	amqpUrl := fs.String("amqp-url", "", "RabbitMQ url")
 	_ = fs.String("config", "", "config file (optional)")
 
+	consulPrefix := fs.String("consul-prefix", "", "DEPRECATED - use --route-prefix")
+	consulMistURL := fs.String("consul-mist-url", "", "DEPRECATED - use --route-mist-url")
```

Comment:
Nice! Should we also have a `--consul` flag like `DEPRECATED - consul is not supported anymore`?

Reply:
If it gets run with that it'll crash with `flag provided but not defined: -consul` — good enough for me, pretty sure that has zero users right now